service/storage_service.hh

```diff
@@ -373,7 +373,7 @@ private:
 
 public:
 
-    static std::list<locator::host_id_or_endpoint> parse_node_list(sstring comma_separated_list);
+    static std::vector<locator::host_id_or_endpoint> parse_node_list(const sstring& comma_separated_list);
```

Comment:
could use the named `locator::host_id_or_endpoint_array` type here? also, `std::string_view` for the arg?